.env.example

```diff
@@ -6,9 +6,22 @@ VITE_DEBUG=true
 DEBUGBAR_ENABLED=false
 
 APP_URL=http://rapidez.test
-MAGENTO_URL=http://localhost:1234
 RAPIDEZ_TOKEN=
 
+# Used by docker-compose to define your published domains
+MAGENTO_HOST="magento.local"
+
+MAGENTO_URL=http://${MAGENTO_HOST}
+# In case you don't want to use traefik
+# MAGENTO_URL=http://127.0.0.1:1234
+
+ELASTICSEARCH_HOST="elasticsearch.local"
+ELASTICSEARCH_PORT=80
+ELASTICSEARCH_USER=elastic
+ELASTICSEARCH_PASS="changeme"
```

Comment:
This pass, and the rapidez pass following it are automatically used while running `docker-compose up`
As are the `MAGENTO_HOST ` and `ELASTICSEARCH_HOST `